packages/@coorpacademy-player-services/src/test/analytics.send-viewed-media-analytics.js

```diff
@@ -1,22 +1,65 @@
+// @flow strict
+
 import test from 'ava';
 import {sendViewedMediaAnalytics} from '../analytics';
+import type {Resource} from '../types';
+
+type DataEvent = {|
+  event: string,
+  mediaType: string,
+  location: string
+|};
+
+// eslint-disable-next-line no-shadow
+type Window = {
+  dataLayer?: Array<DataEvent>
+};
 
-test('populate dataLayer when empty', t => {
+// eslint-disable-next-line no-shadow
+declare var global: {|window: Window|};
```

Comment:
Same as above, this type is defined more than one time. Can you import it from a shared file ?

Reply:
it's not exactly the same depending on tests :S